src/codec_aom.c

```diff
@@ -748,7 +748,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
             cfg->g_lag_in_frames = 0;
         }
         if (encoder->maxThreads > 1) {
-            cfg->g_threads = encoder->maxThreads;
+            // libaom fails if cfg->g_threads is greater than 64 threads. See MAX_NUM_THREADS in
+            // aom/aom_util/aom_thread.h.
+            cfg->g_threads = AVIF_MIN(encoder->maxThreads, 64);
```

Comment:
@wantehchang do you think this fix might make its way to the v1.0.x branch?

Reply:
Frankie: The v1.0.x branch is used for bug-fix patch releases. Since this pull request is not a bug fix, it is more suitable for the next feature release (v1.1.0). However, we can consider making an exception based on customer requests.

Note: This missed the v1.0.4 release that was announced today.

Reply:
That makes sense to me, insofar as this commit changes an argument to avifenc. But I'd argue that the line I commented on here _is_ in fact a bug fix. If you were to run `avifenc --jobs 96 ...` with the aom encoder using v1.0.4 it fails with the error

> aom_codec_enc_init() failed: Invalid parameter: g_threads out of range [..MAX_NUM_THREADS]

but it succeeds on the main branch. This came to my attention because pillow-avif-plugin sets maxThreads, by default, to the number of cores, and users with 64+ cores were unable to encode using AOM (see https://github.com/fdintino/pillow-avif-plugin/pull/41 )

Reply:
Ah, I am sorry I misunderstood you. I thought by "this fix" you meant "this pull request".

Yes, we can cherry-pick just this fix to the v1.0.x branch in preparation for the next patch release.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2008

Reply:
Thank you very much, Vincent!